en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -205,13 +205,33 @@ On the RPi side:
 
 ## Ethernet Connection (Optional)
 
-The flight controller module is [internally connected to RPi CM4](#rpi-cm4-fc-serial-connection) from `TELEM2` (Serial).
+The flight controller module is [internally connected to RPi CM4](#rpi-cm4-fc-serial-connection) from `TELEM2` (serial).
 
 You can also set up a local Ethernet connection between them using the supplied cable.
 Ethernet connectivity provides a fast, reliable, and flexible communication alternative to using USB or other serial connections.
 
 ::: info
-For more general information see: [PX4 Ethernet Setup](../advanced_config/ethernet_setup.md).
+For general Ethernet setup information see: [PX4 Ethernet Setup](../advanced_config/ethernet_setup.md).
+
+The setup here is much the same, except that we have used the following `netplan` config on PX4:
+
+```sh
+network:
+  version: 2
+  renderer: NetworkManager
+  ethernets:
+    eth0:
+      addresses:
+        - 10.41.10.1/24
+      nameservers:
+        addresses: [10.41.10.1]
+      routes:
+        - to: 10.41.10.0/24  # Local route to access devices on this subnet
+          scope: link        # Scope link to restrict it to local subnet
+```
+
+This sets `eth0` as our channel for the local Ethernet link.
+Note that we only set up this locally linked route to prevent internet access disruption over WiFi.
```

Comment:
```suggestion
This sets `eth0` as our channel for the local Ethernet link (instead of ``, which is used in [the normal setup](../advanced_config/ethernet_setup.md#ubuntu-ethernet-network-setup).
Note that we only set up this locally linked route to prevent internet access disruption over WiFi.
```